controllers/springbootapplication_controller.go

```diff
@@ -63,51 +56,6 @@ func SpringBootApplicationReconciler(c controllers.Config, registry cnb.Registry
 	}
 }
 
-func SpringBootApplicationResolveLatestImage(c controllers.Config) controllers.SubReconciler {
-	c.Log = c.Log.WithName("ResolveLatestImage")
-
-	return &controllers.SyncReconciler{
-		Sync: func(ctx context.Context, parent *mononokev1alpha1.SpringBootApplication) error {
-			ref := parent.Spec.ImageRef
-
-			if ref == nil {
-				parent.Status.LatestImage = parent.Spec.Template.Spec.Containers[0].Image
-				return nil
-			}
```

Comment:
This still needs to happen somewhere. Alternatively, we should remove this status field and use `parent.Spec.Template.Spec.Containers[0].Image` to resolve image metadata.

Reply:
dropped the status latestImage property and using the container image directly.